src/database/schema/ChatSchema.ts

```diff
@@ -147,7 +146,7 @@ class ChatSchema implements BaseDbSchema {
           ELSE 0 END AS is_continuous
       FROM 
       ${ChatSchema.getTableName()} A 
-      INNER JOIN ${UserSchema.getTableName()} B 
+      LEFT JOIN ${UserSchema.getTableName()} B 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from an INNER JOIN to a LEFT JOIN can have significant implications on the result set of the query. An INNER JOIN only returns records where there is a match in both tables, while a LEFT JOIN returns all records from the left table and matched records from the right table. If there is no match, the result is NULL on the right side. Ensure that this change aligns with your intended functionality.


<!-- This is an auto-generated comment by OSS CodeRabbit -->